src/node_lib.h

```diff
@@ -352,34 +435,21 @@ NODE_EXTERN v8::MaybeLocal<v8::Value> Call(
     v8::Local<v8::Function> function,
     const std::vector<v8::Local<v8::Value>>& args = {});
 
-NODE_EXTERN v8::MaybeLocal<v8::Value> Call(
-    Environment* env,
-    v8::Local<v8::Object> receiver,
-    v8::Local<v8::Function> function,
-    const std::vector<v8::Local<v8::Value>>& args = {});
-
 /**
  * @brief Calls a given method on a given object.
  *
  * Calls a given method on a given object.
  * Additionally, a list of parameters is passed to the called function.
+ * @param env The environment where this call should be executed.
  * @param object The receiver of the given function.
  * @param function The function to be called.
- * @param args The parameters to pass to the called function. The amount of
- * arguments must be known at compile time.
+ * @param args The parameters to pass to the called function.
  * @return The return value of the called function.
  */
-NODE_EXTERN v8::MaybeLocal<v8::Value> Call(
-    v8::Local<v8::Object> receiver,
-    v8::Local<v8::Function> function,
-    std::initializer_list<v8::Local<v8::Value>> args);
-
 NODE_EXTERN v8::MaybeLocal<v8::Value> Call(
     Environment* env,
     v8::Local<v8::Object> receiver,
     v8::Local<v8::Function> function,
-    std::initializer_list<v8::Local<v8::Value>> args);
-}  // namespace node
```

Comment:
What happened here?

Reply:
Sorry, that was an accident, fixed it :+1:

Reply:
I was just working on the same thing when you started pushing :-D I've now also pushed my commit, which should make this ready.